tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/clean_height_weight_test.py

```diff
@@ -8,13 +66,259 @@
+    CleanHeightAndWeight,)
+from common import JINJA_ENV
+# Project Imports
+from common import MEASUREMENT
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import \
+    BaseTest
+
+# The existing person table is created and partitioned on the pseudo column _PARTITIONTIME,
+# partitioning by _PARTITIONTIME doesn't work using a query_statement for creating a table,
+# therefore CREATE OR REPLACE TABLE doesn' work and we need to DROP the table first.
+MEASUREMENT_DATA_TEMPLATE = JINJA_ENV.from_string("""
+DROP TABLE IF EXISTS
+  `{{project_id}}.{{dataset_id}}.measurement`;
+CREATE TABLE
+  `{{project_id}}.{{dataset_id}}.measurement` AS (
+  WITH
+    w AS (
+    SELECT
+      ARRAY<STRUCT<measurement_id INT64,
+      person_id INT64,
+      measurement_concept_id INT64,
+      measurement_date DATE,
+      measurement_datetime TIMESTAMP,
+      measurement_type_concept_id INT64,
+      operator_concept_id INT64,
+      value_as_number FLOAT64,
+      value_as_concept_id INT64,
+      unit_concept_id INT64,
+      range_low FLOAT64,
+      range_high FLOAT64,
+      provider_id INT64,
+      visit_occurrence_id INT64,
+      measurement_source_value STRING,
+      measurement_source_concept_id INT64,
+      unit_source_value STRING,
+      value_source_value STRING>>
+      [(1,1,3036277,"2017-03-25","2017-03-25 01:00:00 UTC",44818701,4172703,5.6,0,9330,null,null,null,1,"8302-2",3036277,"in",""),
+        (2,2,3036277,"2017-10-18","2017-10-18 01:00:00 UTC",44818701,4172703,62.5,0,9330,null,null,null,2,"8302-2",3036277,"in",""),
+        (3,3,3036277,"2017-02-07","2017-02-07 01:00:00 UTC",44818701,4172703,1.8,null,8582,null,null,null,3,"8302-2",3036277,"cm",""),
+        (4,4,3036277,"2017-01-06","2017-01-06 01:00:00 UTC",44818701,4172703,193.0,0,8582,null,null,null,4,"8302-2",3036277,"cm",""),
+        (5,5,3036277,"2018-01-11","2018-01-11 01:00:00 UTC",44818701,4172703,165.0,0,8582,null,null,null,5,"8302-2",3036277,"cm",""),
+        (6,5,3036277,"2018-02-07","2018-02-07 01:00:00 UTC",44818701,4172703,165.0,0,8582,null,null,null,6,"8302-2",3036277,"cm",""),
+        (7,6,3036277,"2018-11-14","2018-11-14 01:00:00 UTC",44818701,4172703,170.18,0,8582,null,null,null,7,"8302-2",3036277,"cm",""),
+        (8,6,3036277,"2017-05-20","2017-05-20 01:00:00 UTC",44818701,4172703,210.0,0,8582,null,null,null,8,"8302-2",3036277,"cm",""),
+        (9,7,3036277,"2019-08-20","2019-08-20 01:00:00 UTC",44818701,4172703,120,0,8582,null,null,null,9,"8302-2",3036277,"cm",""),
+        (10,8,3036277,"2018-09-10","2018-09-10 01:00:00 UTC",44818701,4172703,125,0,8582,null,null,null,10,"8302-2",3036277,"cm",""),
+        (11,9,3036277,"2017-01-11","2017-01-11 01:00:00 UTC",44818701,4172703,173.0,0,8582,null,null,null,11,"8302-2",3036277,"cm",""),
+        (12,10,3036277,"2018-02-10","2018-02-10 01:00:00 UTC",44818701,4172703,189.0,0,8582,null,null,null,12,"8302-2",3036277,"cm",""),
```

Comment:
It would be good to add `m` and `ft` rows in addition to `cm` and `in` to see how they are corrected.

Reply:
the 1,2,3 rows one each of meters ft and cm.

Reply:
I see, thanks!